server/models/Request.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require("mongoose");
+
+const RequestSchema = new mongoose.Schema({
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "User",
+  },
+  sitterId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "User",
+  },
+  startDate: {
+    type: Date,
+    required: true,
+  },
+  endDate: {
+    type: Date,
+    required: true,
+  },
+  accepted: {
+    type: Boolean,
+    required: true,
+    default: false,
+  },
+  declined: {
+    type: Boolean,
+    required: true,
+    default: false,
+  },
+  paid: {
+    type: Boolean,
+    required: true,
+    default: false,
+  },
```

Comment:
You can also add the mongoose `timestamps` option to tell Mongoose to automatically manage createdAt and updatedAt properties on the document.